Resources/Prototypes/Nyanotrasen/Entities/Structures/Machines/vending_machines.yml

```
@@ -72,7 +72,6 @@
   - type: WiresVisuals
   - type: AccessReader
     access: [["Service"]]
-
 - type: entity
```

It's usually a good idea to keep these separated.
```suggestion

- type: entity
```